### ISSUE TITLE
Adding a dark mode?

### ISSUE THREAD

**Issue by GuillouTran** [RyanWarner/next-mdx-digital-garden-starter#17]
As of 2020, we could probably consider adding a dark mode?

**Comment by RyanWarner** [comment]
Yes, I agree theming would be a great feature. I have a dark mode designed, but it's not yet implemented. Soon!
